nUpdate.Shared/Updating/UpdateManager.cs

```diff
@@ -158,6 +158,11 @@ public partial class UpdateManager : IDisposable
         /// </summary>
         public bool IncludeCurrentPcIntoStatistics { get; set; } = true;
 
+        /// <summary>
+                /// The conditions if an update should be loaded if conditions were specified in the package
```

Comment:
Please modify the docs here: "Gets or sets the additional conditions that determine whether an update should be loaded or not."